src/editors/data/services/cms/api.ts

```diff
@@ -239,82 +333,4 @@ export const apiMethods = {
   ),
 };
 
-export const loadImage = (imageData) => ({
-  ...imageData,
-  dateAdded: new Date(imageData.dateAdded.replace(' at', '')).getTime(),
-});
-
-export const loadImages = (rawImages) => camelizeKeys(rawImages).reduce(
-  (obj, image) => ({ ...obj, [image.id]: module.loadImage(image) }),
-  {},
-);
-
-export const processVideoIds = ({
-  videoId,
-  videoUrl,
-  fallbackVideos,
-}) => {
-  let youtubeId = '';
-  const html5Sources = [];
-
-  if (videoUrl) {
-    if (module.parseYoutubeId(videoUrl)) {
-      youtubeId = module.parseYoutubeId(videoUrl);
-    } else {
-      html5Sources.push(videoUrl);
-    }
-  }
-
-  if (fallbackVideos) {
-    fallbackVideos.forEach((src) => (src ? html5Sources.push(src) : null));
-  }
-
-  return {
-    edxVideoId: videoId,
-    html5Sources,
-    youtubeId,
-  };
-};
-
-export const isEdxVideo = (src) => {
-  const uuid4Regex = /^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}$/;
-  if (src && src.match(uuid4Regex)) {
-    return true;
-  }
-  return false;
-};
-
-export const parseYoutubeId = (src) => {
-  const youtubeRegex = /^((?:https?:)?\/\/)?((?:www|m)\.)?((?:youtube\.com|youtu.be))(\/(?:[\w-]+\?v=|embed\/|v\/)?)([\w-]+)(\S+)?$/;
-  if (!src.match(youtubeRegex)) {
-    return null;
-  }
-  return src.match(youtubeRegex)[5];
-};
-
-export const processLicense = (licenseType, licenseDetails) => {
-  if (licenseType === 'creative-commons') {
-    return 'creative-commons: ver=4.0'.concat(
-      (licenseDetails.attribution ? ' BY' : ''),
-      (licenseDetails.noncommercial ? ' NC' : ''),
-      (licenseDetails.noDerivatives ? ' ND' : ''),
-      (licenseDetails.shareAlike ? ' SA' : ''),
-    );
-  }
-  if (licenseType === 'all-rights-reserved') {
-    return 'all-rights-reserved';
-  }
-  return '';
-};
-
-export const checkMockApi = (key) => {
-  if (process.env.REACT_APP_DEVGALLERY) {
-    return mockApi[key] ? mockApi[key] : mockApi.emptyMock;
-  }
-  return module.apiMethods[key];
-};
-
-export default Object.keys(apiMethods).reduce(
-  (obj, key) => ({ ...obj, [key]: checkMockApi(key) }),
```

Comment:
This mock API was for the preview server that existed in `frontend-lib-content-components`. Last time I tried it, it wasn't working at all, and now that we've moved the code to frontend-app-authoring, it doesn't even exist. So I deleted the unused mockApi code.